bound/bound.go

```diff
@@ -20,6 +20,13 @@ func (b Bound) IsValid() bool {
 	return b.Min <= b.Max
 }
 
+// Jaccard returns a jaccard index between bounds
+func Jaccard(bounds ...Bound) float64 {
```

Comment:
Sorry I missed this before. This should only have two input params, `Jaccard(a, b Bound) float64`. A distance between more than that is not meaningful.

Reply:
Are you sure about this? The Jaccard wikipedia article defines it as "The intersection of a set divided by the union of a set". That seems pretty well defined, and this package already has a definition of intersection and union.

Reply:
You're right. Thinking about it formally, I could see a meaning, as soon as I thought about it visually, it makes sense.